pkg/minikube/console/style.go

```diff
@@ -131,6 +132,11 @@ func lowPrefix(s style) string {
 // Apply styling to a format string
 func applyStyle(style string, useColor bool, format string, a ...interface{}) (string, error) {
 	p := message.NewPrinter(preferredLanguage)
+	for i, x := range a {
+		if _, ok := x.(int); ok {
+			a[i] = number.Decimal(x, number.NoSeparator())
```

Comment:
Sneaky. Let's add a test so that this doesn't accidentally get refactored out in a 
 future PR.

Reply:
Done